frame/staking/src/lib.rs

```diff
@@ -586,9 +643,24 @@ impl<T: Trait> SessionInterface<<T as frame_system::Trait>::AccountId> for T whe
 		<pallet_session::Module<T>>::validators()
 	}
 
+	fn keys<KeyT: RuntimeAppPublic + Decode + sp_std::fmt::Debug>()
```

Comment:
@tomusdrw  this is part of the temporal remedy of not using `SubmitSignedTransaction`